lms/djangoapps/instructor_task/tests/test_base.py

```diff
@@ -384,9 +384,11 @@ def _extract_and_round_numeric_items(dictionary):
         """
         extracted = {}
         for key, value in dictionary.items():
+            if value is None:
```

Comment:
Why is this needed,  it might be that we're now getting none that we didn't used to before so it may be worth digging into what's going on upstream.

Reply:
`float` doesn't allow `None` as a parameter, yes we might need to look into test data having a `None` in the first place

Reply:
Ok, please look into that because `float(None)` should be throwing this error in python 2 as well so if it's not it might indicate that something is wrong upstream.

Reply:
I've removed the condition and fixed the `None` test data causing the issue